liquibase-service/src/main/resources/db/changelog/db.rdb_modern.changelog-16.1.yaml

```diff
@@ -142,90 +142,103 @@ databaseChangeLog:
   - changeSet:
       id: 20
       author: liquibase
+      runOnChange: true
```

Comment:
Would there be any issues if this is enabled for all the scripts?

Reply:
Not really, as long as we handle the create scripts with `if exists` clause. Otherwise we will end up with SQL server duplicate object exception..

Reply:
> Would there be any issues if this is enabled for all the scripts?

It will run the changeset only if underlying script is chamged. It won't affect existing scripts if none is changed.